plugins/widget/plugin.js

```diff
@@ -1183,12 +1185,14 @@
 						that.fire( 'data', that.data );
 						that.editor.fire( 'saveSnapshot' );
 					}
-				} );
+				}, null, null, 0 );
```

Comment:
Probably the priority of this listener shouldn't be modified as it basically changes the state of the editor. The simplest solution would be to introduce dedicated `onOk` handler to add appropriate data to the event.